wsmaster/che-core-api-metrics/src/main/java/org/eclipse/che/api/metrics/WorkspaceFailureMeterBinder.java

```diff
@@ -50,10 +51,11 @@ public void bindTo(MeterRegistry registry) {
     // only subscribe to the event once we have the counters ready
     eventService.subscribe(
         event -> {
-          if (event.getError() == null || event.getStatus() != WorkspaceStatus.STOPPED) {
+          if (event.getError() == null
```

Comment:
just to confirm: `event.getError()` returns non `null` value if a user explicitly stops workspace during startup?

Reply:
In that case, error would have a value `"Runtime start for identity 'workspace: %s, environment: %s, ownerId: %s' is interrupted"`

Reply:
and `event.isInitiatedByUser=true`